README.md

```diff
@@ -1208,12 +1208,12 @@ PersonMapper.to_json(person, pretty: true)
 
 Shale supports these types out of the box:
```

Comment:
Can you add a section to the readme, describing that passing a class as a type is also valid, with some small example?

Reply:
I extended this section to mention that the symbol alias and the class are interchangeable and included a small example. I did the same with the "Writing your own type" section. Let me know if that looks sufficient for you or if you'd like it somewhere else.

Reply:
perfect, thank you